CONTRIBUTING.md

```diff
@@ -30,114 +28,136 @@ Before you get started, please read and follow these guidelines to ensure a smoo
 
 ### Suggesting enhancements
-
 If you have ideas for enhancements or new features, feel free to create an issue to discuss them. Gravitino welcomes suggestions and provides prompt feedback on their feasibility and relevance.
-
 ### Good first issues
-
 If you are new to open source or can't find something to work on check out the [Good First Issues list](https://github.com/datastrato/gravitino/contribute).
-
 ### Working on issues
-
 Check out the list of open issues and find one that interests you. You can also comment on an issue to indicate that you're working on it. Please keep the issue updated with your progress.
-
 ## Creating pull requests
-
 Create a new branch from `main` for your changes:
-
 ```bash
 git checkout -b your-branch-name
 ```
-
 Make your changes and commit them. Be sure to write clear and concise commit messages.
-
 ```bash
 git commit -m "Your commit message"
 ```
-
 Push your changes to your fork on GitHub:
-
 ```bash
 git push your-branch-name
 ```
-
 After you have pushed your changes, create a pull request (PR) in the Gravitino repository. Be sure to provide a detailed description of your changes, reference any related issues and please follow the template provided. Gravitino's maintainers evaluate pull requests, offer feedback, and assist in merging project changes.
-
 ### The Review Process
-
 For details on the review process, please refer to [MAINTAINERS.md](MAINTAINERS.md).
-
 ## Testing
-
 The CI infrastructure runs unit and integration tests on each pull request, please make sure these tests pass before making a pull request.
 
 The unit tests run on every build and integration tests run as needed. See [how to test](docs/how-to-test.md) for more information.
 
 When adding new code or fixing a bug be sure to add unit tests to provide coverage.
-
 ## Coding standards
-
 Spotless checks code formatting. If your code isn't correctly formatted, the build fails. To correctly format your code please use Spotless.
-
 ```bash
 ./gradlew spotlessApply
 ```
-
 All files must have a license header and the build fails if any files are missing license headers. If you are adding third-party code be sure to understand how to add the third-party license to Gravitino LICENSE and NOTICE files.
 
 For any bugs or new code please add unit tests to provide coverage of the code. The project may not accept code without unit tests.
 
 All text files should use macOS/unix style line endings (LF) not windows style line endings (CRLF).
-
 ## Community and communication
-
 Join the [community discourse group](https://gravitino.discourse.group) to discuss ideas and seek help. You are also encouraged to use GitHub discussions and follow Datastrato on social media to stay updated on project news.
-
 ## License
-
```

Comment:
Hi @LanceHsun please avoid deleting the necessary blank line. What I mean previously is that only **one** blank line is enough, but the blank line is necessary, don't delete it.

Reply:
ok, done!